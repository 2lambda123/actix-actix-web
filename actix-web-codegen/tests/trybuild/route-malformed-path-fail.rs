use actix_web_codegen::*;

#[get("/one/{", other)]
async fn one() -> String {
    "Hello World!".to_owned()
}
fn main() {}
