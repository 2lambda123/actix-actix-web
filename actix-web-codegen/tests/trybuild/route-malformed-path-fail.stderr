error: Unknown attribute.
 --> $DIR/route-malformed-path-fail.rs:3:17
  |
3 | #[get("/one/{", other)]
  |                 ^^^^^
